Code/Editor/EditorViewportSettings.cpp

```diff
@@ -588,4 +605,15 @@ namespace SandboxEditor
     {
         AzToolsFramework::SetRegistry(CameraFarPlaneDistanceSetting, aznumeric_cast<double>(distance));
     }
+
+    float CameraDefaultFovRadians()
+    {
+        return aznumeric_caster(AzToolsFramework::GetRegistry(CameraFovSetting, 90.0));
+    }
+
+    void SetCameraDefaultFovRadians(float fov)
```

Comment:
update: Please update parameter name to `fovRadians`

Reply:
we should also have a function that sets that value in degrees and calls DegToRad/RadToDeg (we should really store the value in the settings registry as degrees too as this will be much easier for customers to understand)